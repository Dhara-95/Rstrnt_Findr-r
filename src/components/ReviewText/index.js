import React from "react";

const ReviewText = (props) => {
  const { text } = props;

  return <p className="text-grey text-xl text-center font-bold">{text}</p>;
};

export { ReviewText };
