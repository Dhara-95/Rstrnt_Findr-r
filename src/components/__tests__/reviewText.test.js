import React from "react";
import renderer from "react-test-renderer";

import { ReviewText } from "../ReviewText";

it("displays the reviewe text correctly", () => {
  const tree = renderer.create(<ReviewText text={"text"} />).toJSON();
  expect(tree).toMatchSnapshot();
});
