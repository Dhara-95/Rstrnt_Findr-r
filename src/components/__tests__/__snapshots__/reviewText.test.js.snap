// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the reviewe text correctly 1`] = `
<p
  className="text-grey text-xl text-center font-bold"
>
  text
</p>
`;
